ivy/functional/backends/paddle/experimental/elementwise.py

```diff
@@ -200,7 +224,10 @@ def diff(
     append: Optional[Union[paddle.Tensor, int, float, list, tuple]] = None,
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
-    raise IvyNotImplementedException()
+    x=paddle.to_tensor(x)
+    if out is not None:
```

Comment:
The paddle library **diff** function has an argument   **name (str, optional)** . so i have not used **out** argument.